explorer/app/models/responses.ts

```diff
@@ -193,5 +202,5 @@ export interface SummaryResponse {
   }[];
   speciesCount: number;
   specimenCount: number;
-  totalFileSize: number;
+  totalFileSize: string | number;
```

Comment:
Can you tell me why this might be a string? Are you seeing both string and number values in the response from Azul?

Reply:
I haven't seen a string response, but used the special handling from the Angular version to assume that it could be string or number.